lxc/profile.go

```diff
@@ -123,8 +125,8 @@ func (c *profileCmd) run(config *lxd.Config, args []string) error {
 		default:
 			return errArgs
 		}
-		return c.doProfileApply(client, container, profile)
-	case "apply-add":
+		return c.doProfileAssign(client, container, profile)
+	case "add":
```

Comment:
On Mon, May 23, 2016 at 06:07:13AM -0700, ctrlrsf wrote:
> > @@ -123,8 +125,8 @@ func (c *profileCmd) run(config *lxd.Config, args []string) error {
> >  		default:
> >  			return errArgs
> >  		}
> > -		return c.doProfileApply(client, container, profile)
> > -	case "apply-add":
> > +		return c.doProfileAssign(client, container, profile)
> > +	case "add":
> 
> Should this case also catch "apply-add" to remain backwards compatible?

apply-add isn't in any released version of lxd, so I think it's fine.

> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/2038/files/3a6d6a56d214f7b212d283f6c7ba992ef99e618f#r64218511